batch/test/utils.py

```diff
@@ -18,3 +18,10 @@ def smallest_machine_type(cloud):
         return 'n1-standard-1'
     assert cloud == 'azure'
     return 'Standard_D2ds_v4'
+
+
+def nondefault_region(cloud):
```

Comment:
If this is only used in one test can we just keep it in that test? Also this is a nit because I don't see us changing our default region to `us-east1`, but it does bother me slightly that it's not necessarily correct. We could make it correct by defining two regions and picking one that is not the default region.